docs/bootstrap.md

```diff
@@ -11,6 +11,18 @@ permalink: /docs/bootstrap/
 
 The `alt.bootstrap()` function takes in a snapshot you've saved and reloads all the state with that snapshot, no events will be emitted to your components during this process, so it is advised to do this on init before the view has even rendered.
 
+If you need to emit a change event, you can use `this.emitChange` inside of your `bootstrap` life cycle method.
```

Comment:
If you want to emit a change event after bootstrapping you can use [`this.emitChange`](link to emit change) inside of the [bootstrap lifecycle method](lifecycle.md).